org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/AbstractADWindowContent.java

```diff
@@ -1113,7 +1133,7 @@ public void onToggle()
     	adTabbox.getSelectedTabpanel().switchRowPresentation();
     	//Deepak-Enabling customize button IDEMPIERE-364
         if(!(adTabbox.getSelectedTabpanel() instanceof ADSortTab))
-        	toolbar.enableCustomize(((ADTabpanel)adTabbox.getSelectedTabpanel()).isGridView());
+        	toolbar.enableCustomize(adTabbox.getSelectedTabpanel().isGridView());
```

Comment:
shouldn't this be replaced by the new isEnableCustomizeButton() method ?

Reply:
Agree. Changing.

Reply:
Done